athena-storage-api/pom.xml

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>aws-athena-query-federation</artifactId>
+        <groupId>com.amazonaws</groupId>
+        <version>2022.42.2</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>athena-storage-api</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <aws-athena-federation-sdk.version>2022.2.1</aws-athena-federation-sdk.version>
+        <argLine>
+            -Djdk.attach.allowAttachSelf=true
+            --add-opens java.base/jdk.internal.loader=ALL-UNNAMED
+        </argLine>
+        <hadoop.version>2.10.1</hadoop.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>libraries-bom</artifactId>
+                <version>25.2.0</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-storage</artifactId>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.arrow/arrow-vector -->
+        <dependency>
+            <groupId>org.apache.arrow</groupId>
+            <artifactId>arrow-vector</artifactId>
+            <version>3.0.0</version>
```

Comment:
You should not be using 3.0.0 anymore. The rest of the federation github repository has already been upgraded to using 9.0.0.

Please make use of the ${apache.arrow.version} property as well.

Reply:
Used ${apache.arrow.version} instead